designs/flowehr-oss.md

```diff
@@ -0,0 +1,119 @@
+
+This does mean that FlowEHR's dedicated maintainers will need to stay on top of new feature requests, bugs and PRs. As these could come from anywhere, rigour around management and quality control will be essential.
+
+### 2. How will they contribute?
+
+Some contributors may wish to both raise a request for and develop new features (MerseyCare, the Microsoft team, etc.) and others may wish to do one or the other. The following proposed flow assumes the whole journey from request to merge is done by a single contributor, but some requesters may not develop features themselves, and it's up to maintainers to priotise these and task out the work to other contributors.
+
+> The following flow also applies to bugs
+
+![Contributor flow](./assets/contributing.png)
+
+1. Contributor raises an issue in the repo to describe the desired feature
+    - On opening the issue, the contributor should be prompted to complete a template to describe the feature
+1. On opening, the issue is automatically tagged with a `needs: triage` label to make it easier to filter issues that need maintainer attention
+1. The maintainer reviews the issue, removes the traige label, and either:
+    - Accepts the issue and deems no design to be necessary (typically smaller features/bug fixes)
+    - Accepts the issue and adds a `needs: design` label to indicate that design is required (typically larger features where a better idea of implementation is needed)
+    - Prompts the contributor for more information
+    - Rejects the issue and closes it
+1. If a design is needed, a design document should be added to the issue body under a design heading. We should provide a template for this to capture expected design elements
+    - Alternatively bigger designs may be captured in a separate document (in the repo in a `/designs` folder) and linked to from the issue
+1. The maintainer reviews the design and either:
+    - Accepts the design and removes the `needs: design` label
+    - Prompts the contributor for more information/works with them to refine
+    - Rejects the design &/or closes the issue
+1. The maintainer and contributor work together to break down the feature into smaller stories if neccessary
+1. The contributor forks the repo and develops the feature in their own fork
+1. The contributor raises a PR to merge the feature into the repo
+1. Maintainer(s) review the PR, kick off automated tests if needed, and either:
+    - Accept the PR and merge it into the main repo
+    - Request changes
+    - Reject the PR and close it
+
+### Questions
+
+- Who will be the maintainers? Will this be a team within UCLH? Joint between Merseycare and UCLH (& Msft?)?
+    - Who will review new items in the backlog?
+    - Who will review designs?
+    - Who will review PRs (code)?
+    - Who will manage releases?
+    - Who will monitor/fix CI/CD environments?
+- How many PR reviews will be needed by maintainers?
```

Comment:
Had a quick look and couldn't find it existing already, but can we create an issue for the backlog to resolve these questions? 🙏 

This may also feed into the need for upskilling or training for the roles

The questions further on feel like they may be worked out more iteratively, but it'd be good to explicitly address these ones I think, though open to other suggestions